Dockerfile.build

```diff
@@ -21,8 +21,7 @@ RUN go get -u github.com/golang/dep/cmd/dep \
 && go get -v -u github.com/markbates/filetest \
 && go get -v -u github.com/markbates/grift \
 && go get -v -u github.com/markbates/refresh \
-&& go get -u github.com/alecthomas/gometalinter \
-&& gometalinter --install
+&& go get -v -u github.com/golangci/golangci-lint/cmd/golangci-lint
```

Comment:
Can we use the curl version instead here (since we already have curl in this image)? It will be faster to build and won't download unneeded sources. :)